lib/ruby/signature/prototype/rb.rb

```diff
@@ -322,13 +322,112 @@ def function_type_from_body(node)
           fun
         end
 
+        def function_return_type_from_body(node)
+          body = node.children[2]
+          return Types::Bases::Nil.new(location: nil) unless body
+
+          literal_to_type(body)
+        end
+
+        def literal_to_type(node)
+          case node.type
+          when :STR
+            lit = node.children[0]
+            if lit.match?(/\A[ -~]+\z/)
+              Types::Literal.new(literal: lit, location: nil)
+            else
+              Types::ClassInstance.new(name: '::String', args: [], location: nil)
```

Comment:
How about using `Ruby::Signature::BuiltinNames::String.instance_type` ?

Reply:
Thanks for the suggestion! I didn't know that.

I replaced them with BuiltinNames. 773015a4cc99419994b29b67d44a52ef57c90c4f